reference-implementation/lib/parser.js

```diff
@@ -62,6 +62,10 @@ function sortAndNormalizeSpecifierMap(obj, baseURL) {
               `Package address targets must end with "/".`);
           return false;
         }
+        if (address.protocol === BUILT_IN_MODULE_PROTOCOL && address.href.includes('/')) {
+          console.warn(`Invalid target address "${address.href}". Built-in module URLs must not contain "/".`);
```

Comment:
I'm not sure about the outright banning of `/`, since it seems useful to be able to define names like `std:[spec]/[feature]`, which can be supported absolutely fine without running into subpath issues. It seems fine if this is just while these cases are being worked out, but I'm weary of this being enforced.

Reply:
I agree this would be nice. I'll file a follow-up issue, and maybe you or someone else can tackle fixing it :).